### PR TITLE
Support unknown bus IDs

### LINKED ISSUE #181: Full support for unknown bus IDs
We should be able to handle unknown bus IDs in for manual Board Bus.

### PR DESCRIPTION
**Pull request by Gerzer**
Closes #181

### CODE REVIEW

File: Shared/Models/Bus.swift [hunk 1]
```diff
@@ -72,7 +76,7 @@ class Bus: NSObject, Codable, Identifiable, CustomAnnotation {
 			case .system:
 				return AppStorageManager.shared.colorBlindMode ? .purple : .red
 			case .user:
-                return self.id>0 ? .green: AppStorageManager.shared.colorBlindMode ? .purple : .red
+				return self.id > 0 ? .green : (AppStorageManager.shared.colorBlindMode ? .purple : .red)
```
Comment (tommytrg297) [inline Shared/Models/Bus.swift:75]: if self.id > 0 { 
return .green}
else { ... } 

so it's coherent with line 55 where you use if else

File: Shared/Views/BusOption.swift [hunk 2]
```diff
@@ -25,7 +25,7 @@ struct BusOption: View {
 				}
 			}
 		} label: {
-			Text("\(self.busID.rawValue)")
+			Text(self.busID.isUnknown ? "I Don’t Know" : "\(self.busID.rawValue)")
```
Comment (tommytrg297) [inline Shared/Views/BusOption.swift:28]: no caps "I don't know"

File: Shared/Views/MapContainer.swift [hunk 3]
```diff
@@ -68,7 +68,8 @@ struct MapContainer: View {
 			}
 			#if os(iOS)
 			if case .some(.onBus) = self.travelState, let busID = self.busID, let coordinate = CLLocationManager.default.location?.coordinate {
-				Marker("Bus \(busID)", systemImage: SFSymbol.user.systemName, coordinate: coordinate)
+				let title = if busID > 0 { "Bus \(busID)" } else { "Bus" }
```
Comment (tommytrg297) [inline Shared/Views/MapContainer.swift]: let title = busID > 0 ? "Bus \(busID)" : "Bus"

**Review by tommytrg297 — COMMENTED** [review]
added some comments

File: Shared/Views/BusOption.swift (hunk 2 above)
Comment (Gerzer) [reply Shared/Views/BusOption.swift:28]: Sorry, overruled! 🔨☭

Comment (Gerzer) [reply Shared/Views/BusOption.swift:28]: Check literally any first-party app lol

**Review by tommytrg297 — APPROVED** [review]

**Pull request merged**